ruff.toml

```diff
@@ -0,0 +1,5 @@
+line-length = 120
+
+[lint]
+select = ["C", "E", "F", "R", "W", "I", "N", "Q", "UP", "RUF", "ISC", "T20"]
```

Comment:
Never used ruff before, but it's a shame it uses such cryptic abbreviations for its lints (unlike the linters I'm used to). Could we have comments explaining what all of these are?

Reply:
I've tried going down that comment route before. It wasn't good.

If you have a good editor, you can just install this extension https://github.com/Jannchie/ruff-ignore-explainer
Then it shows you the more clear names without having to maintain the comments.